content/client-server-api/modules/report_content.md

```diff
@@ -29,3 +29,20 @@ is in before accepting a report.
 based on whether or not the reporting user is joined to the room. This is
 because users can be exposed to harmful content without being joined to a
 room. For instance, through room directories or invites.
+
+{{% added-in v="1.14" %}} Similarly, servers MUST NOT restrict user reports
+based on whether or not the reporting user is joined to any rooms that the
+reported user is joined to. This is because users can be exposed to harmful
+content without being joined to a room. For instance, through user
+directories or invites.
+
+Clients can infer whether a reported event, room or user exists based on the
+404 responses of the reporting endpoints. Homeservers that wish to conceal
+this information MAY return 200 responses regardless of the existence of the
+reported subject.
+
+Furthermore, it might be possible for clients to deduce whether a reported
+event, room or user exists by timing the response. This is because only a
+report for an existing subject will require the homeserver to do further
+processing. To combat this, homeserver implementations MAY add a random
+delay when generating a response.
```

Comment:
This has previously not been stated for the room-reporting endpoint. However, I cannot think of a reason why it wouldn't apply there, too. So I'd consider adding it to be a clarification that I'm combining into this spec changes.

Reply:
This would be part of the MSC for changing the report room endpoint.